pkg/services/ngalert/notifier/channels/slack.go

```diff
@@ -118,12 +117,11 @@ func NewSlackNotifier(model *models.AlertNotification, t *template.Template) (*S
 		MentionGroups:  mentionGroups,
 		MentionChannel: mentionChannel,
 		Username:       model.Settings.Get("username").MustString("Grafana"),
-		IconEmoji:      model.Settings.Get("icon_emoji").MustString(),
-		IconURL:        model.Settings.Get("icon_url").MustString(),
+		IconEmoji:      model.Settings.Get("iconEmoji").MustString(),
+		IconURL:        model.Settings.Get("iconUrl").MustString(),
```

Comment:
This is a breaking change for the [provisioning](https://grafana.com/docs/grafana/latest/administration/provisioning/#supported-settings) of notification channels.
Also if we decide to modify the properties we need to make sure we migrate them properly when migrating the existing notification channels.

Reply:
What was confusing to me was we were using `iconEmoji` and `iconUrl` while registering currently (see [here](https://github.com/grafana/grafana/blob/3b515e650c55f0125020660d8aa22811f756b0d3/pkg/services/alerting/notifiers/slack.go#L61-L73)), and use it as `icon_emoji` and `icon_url` instead while accessing the settings. Does that mean it is broken currently? I can change back to using `_` here, no issues.